Lob-API-public.yml

```
@@ -6,6 +6,8 @@ info:
   description: >
     The Lob API is organized around REST. Our API is designed to have predictable,
     resource-oriented URLs and uses HTTP response codes to indicate any API errors.
+      <p>
+    Looking for our [previous documentation](https://lob.github.io/legacy-docs/)?
```

mmm this kind of bothers me
```suggestion
    <p>
    Looking for our [previous documentation](https://lob.github.io/legacy-docs/)?
```